config/config.go

```diff
@@ -23,7 +23,7 @@ func Get() (*Config, error) {
 	}
 
 	cfg := &Config{
-		BindAddr:                   ":",
+		BindAddr:                   ":24400",
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        10 * time.Second,
```

Comment:
I know you inherited this code but do you mind updating this configuration, default HealthCheckInterval should be 30seconds and HealthcheckCriticalTimeout should be 90 seconds

Reply:
Bump

```
                HealthCheckInterval:        30 * time.Second,
                HealthCheckCriticalTimeout: 90 * time.Second,
```